pages/linux/protonvpn-cli.md

```diff
@@ -0,0 +1,36 @@
+# protonvpn-cli
+
+> Official client for ProtonVPN service from the command-line.
+> More information: <https://github.com/ProtonVPN/linux-cli>.
+
+- Log in to the ProtonVPN account:
+
+`protonvpn-cli login {{username}}`
+
+- Start Kill Switch upon connecting to ProtonVPN:
```

Comment:
```suggestion
- Start a Kill Switch upon connecting to ProtonVPN:
```

Reply:
Not sure, `Kill Switch` is the name of the feature, it's not a common standard thing in networking. https://protonvpn.com/support/what-is-kill-switch/

Reply:
I suggest keeping this as-is if it's the name of a technical feature. Sure it's not technically grammatically correct, but it makes more sense to keep it as-is if `Kill Switch` has specific technical meaning.

Reply:
a kill-switch seems to be a normal thing for VPNs to have. However I don't mind, but its more of standard rather a feature, if you understand what I mean

Reply:
Oh, https://us.norton.com/internetsecurity-privacy-vpn-kill-switch-what-is-it-how-does-it-work.html. you're right, then:
```suggestion
- Start a kill switch upon connecting to ProtonVPN:
```